src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Formatting/Rules/BaseIndentationFormattingRule.cs

```diff
@@ -65,44 +67,44 @@ private void AddNextIndentBlockOperations(List<IndentBlockOperation> list, Synta
 
         private void AdjustIndentBlockOperation(List<IndentBlockOperation> list)
         {
-            for (var i = 0; i < list.Count; i++)
-            {
-                var operation = list[i];
-
-                // already filtered out operation
-                if (operation == null)
-                {
-                    continue;
-                }
-
-                // if span is same as us, make sure we only include ourselves.
-                if (_span == operation.TextSpan && !Myself(operation))
-                {
-                    list[i] = null;
-                    continue;
-                }
-
-                // inside of us, skip it.
-                if (_span.Contains(operation.TextSpan))
+            list.RemoveOrTransformAll(
+                (operation, self) =>
```

Comment:
i wondre if we should have a way to mark these (or the RemoveOrTransformAll) as wanting static-lambdas.  THat way when we get static-lambdas we can have somethin verify that this is correct and isn't actually capturing.

Reply:
File an issue on dotnet/roslyn-analyzers for a PerformanceSensitive analyzer?